src/pages/Recipe.test.js

```diff
@@ -15,7 +21,11 @@ describe('<Recipe />', () => {
   });
 
   it('renders correctly', () => {
-    const component = renderer.create(<Recipe {...props('?id=1')} />);
+    const component = renderer.create(
+      <Provider store={store}>
+        <Recipe {...props('?id=1')} />
```

Comment:
You need to pass actions to this guy, sorry if I was unclear earlier when I said not to pass props to it - the only props you'd want to pass would be special props like `location` or `actions` that get injected by a HOC.